source/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter.h

```diff
@@ -50,12 +50,15 @@ class Http1BridgeFilter : public Http::StreamFilter {
   bool doStatTracking() const { return request_names_.has_value(); }
 
 private:
-  void chargeStat(const Http::HeaderMap& headers);
-  void setupStatTracking(const Http::HeaderMap& headers);
+  template <class T> void chargeStat(const T& headers) {
```

Comment:
should this be using `ResponseHeaderOrTrailersMap` instead of being generic? Don't have a strong opinion here, both works

Reply:
This is from before I had that. I will revert this, thanks for the catch. Same below.